charts/placement-policy-scheduler-plugins/templates/rbac.yaml

```diff
@@ -1,8 +1,24 @@
 apiVersion: rbac.authorization.k8s.io/v1
 kind: ClusterRole
 metadata:
-  name: pp-plugins-scheduler
+  name: aggregated-pp-plugins-scheduler-reader
+  labels:
+    rbac.authorization.k8s.io/aggregate-to-view: "true"
+    rbac.authorization.k8s.io/aggregate-to-edit: "true"
+    rbac.authorization.k8s.io/aggregate-to-admin: "true"
 rules:
+- apiGroups: ["*"]
+  resources: ["pods", "nodes"]
+  verbs: ["get", "list", "watch"]
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: system:pp-plugins-scheduler
+rules:
+- apiGroups: [""]
+  resources: ["pods", "nodes", "nodes/stats", "namespaces", "configmaps"]
```

Comment:
`pods` and `nodes` verbs are already defined:

https://github.com/Azure/placement-policy-scheduler-plugins/blob/58e2c66cc171062812b4e36b9f7e589c455a0512/charts/placement-policy-scheduler-plugins/templates/rbac.yaml#L26-L28

https://github.com/Azure/placement-policy-scheduler-plugins/blob/58e2c66cc171062812b4e36b9f7e589c455a0512/charts/placement-policy-scheduler-plugins/templates/rbac.yaml#L23-L25

Do we need `node/stats`?

Reply:
removed "pods", "nodes", "nodes/stats"